Source/Response.swift

```diff
@@ -77,4 +77,16 @@ public extension Response {
         }
         return string
     }
+
+    /// Maps received data at key path into a String.
+    public func mapString(atKeyPath keyPath: String?) throws -> String {
+        guard let keyPath = keyPath else { return try mapString() }
+
+        guard let jsonDictionary = try mapJSON() as? NSDictionary,
+            let string = jsonDictionary.value(forKeyPath:keyPath) as? String else {
+                throw Error.jsonMapping(self)
```

Comment:
Also I'm not sure whether it should be `.jsonMapping` error or `.stringMapping` error. The function name still is `mapString` so I'm more sold on the latter.